README.md

```diff
@@ -1,2 +1,149 @@
-# oop-school-library
- "OOP school library" app to record what books are in the library and who borrows them
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>"OOP school library"</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 ["OOP school library"] <a name="about-project"></a>
+
+**["OOP school library"]** is a tool to record what books are in the library and who borrows them. The app will allow to:
+
+Add new students or teachers.
+Add new books.
+Save records of who borrowed a given book and when.
+And all of this will be built in a beautiful and well-organized way!
+## 🛠 Built With <a name="built-with"></a>
+
+1- Ruby<br>
+2- Gitflow<br>
+3- Rubocop
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.microverse.org/">Microverse</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Add new students or teachers class]**
+- **[Add new person parent class]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming Soon...](https://github.com/beteland123/oop-school-library.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web Browser (Chrome recommended)<br>
+Code editor (VS recommended)<br>
+GitHub account<br>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone "https://github.com/beteland123/oop-school-library.git"
+```
```

Comment:
- [ ] To improve your README looks professional and descriptive, I think it would be better if you improve `Getting Started` section will all information needed by users in order to use your project locally such as how to `access the cloned` directory, `install bundles`(if exist), `run your` app,... Form more details I suggest you this **[gist_file](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)**